packages/firestore/test/util/test_platform.ts

```diff
@@ -272,11 +273,11 @@ export class TestPlatform implements Platform {
   }
 }
 
-/** Returns true if we are running under Node. */
-export function isNode(): boolean {
```

Comment:
Seems like this wasn't being used anywhere, and there's a better version in @firebase/util so I hope it's okay to remove?

Reply:
@wu-hui might have just started using this. He would probably be delighted to switch to the better version :)